src/plugins/localStorage/LocalStorageObjectProvider.js

```diff
@@ -41,6 +41,10 @@ export default class LocalStorageObjectProvider {
         }
     }
 
+    getAllObjects() {
```

Comment:
need access to all the objects as the annotations aren't accessible from walking the tree.